docs/1-introduction/1-1-getting-started/1-1-2-hello-d3d11.md

```diff
@@ -7,38 +7,44 @@ for an application with a window through GLFW. The implementation for `Main.cpp`
 
 If you are looking at the source code for this chapter, you will also notice that `Application.cpp`
 and `Application.hpp` do not exist anymore, as we have moved both of these files into a separate
-`Framework` project to ease development between chapters. This `Framework` project will include
+`Framework` project that creates a static lib, to ease development between chapters. This `Framework` project will include
```

Comment:
I would personally prefer "library" rather than "lib", keeps it a bit more formal and less lazy

Reply:
I concur